test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -208,6 +208,18 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
     return setup(settings);
   }
 
+  function setupPIV (pivAuthentication, useDefaultText) {
+    var buttonText = (useDefaultText) ? '' : 'piv test text';
+    var settings = {
+      piv: {
```

Comment:
Is there a case where there can be a piv config without certAuthUrl ? Should we just not have the piv config if setup(false) is called.

Reply:
certAuthUrl is required for PIV.. it wont know the endpoint to call to prompt for cert otherwise.. I want to be able to test that no certAuthUrl = no piv button, so i wrote it this way